CGCharter-1727386911.html

```diff
@@ -0,0 +1,306 @@
+      </p>
+      
+    <h2 id="deliverables">
+      Deliverables
+    </h2>
+    <h3 id="specifications">
+      Specifications
+    </h3>
+    <p class="remove">
+      {TBD: Provide a brief description of each specification the group plans
+      to produce. Where an estimate is possible, it can be useful to provide an
+      estimated schedule for key deliverables. As described below, the group
+      may later modify the charter deliverables. if no specifications, include:
+      "No Specifications will be produced under the current charter."}
+    </p>
+    <h3 id="non-normative-reports">
+      Non-Normative Reports
+    </h3>
+    <p>
+      The group may produce other Community Group Reports within the scope of
+      this charter but that are not Specifications, for instance use cases,
+      requirements, or white papers.
+    </p>
+    <h3 id="test-suites">
+      Test Suites and Other Software
+    </h3>
+    <p class="remove">
+      {TBD: If there are no plans to create a test suite or other software,
+      please state that and remove the following paragraph. If Github is not
+      being used, then indicate where the license information is. If GitHub is
+      being used link to your LICENSE.md file in the next paragraph.}
+    </p>
+    <p>
+      The group MAY produce test suites to support the Specifications. Please
+      see the GitHub LICENSE file for test suite contribution licensing
+      information.
+    </p>
+    <h2 id="liaisons">
+      Dependencies or Liaisons
+    </h2>
+    <p class="remove">
+      {TBD: List any significant dependencies on other groups (inside or
+      outside W3C) or materials. }
+    </p>
+    <h2 id="process">
+      Community and Business Group Process
+    </h2>
+    <p>
+      The group operates under the <a href=
+      "https://www.w3.org/community/about/process/">Community and Business
+      Group Process</a>. Terms in this Charter that conflict with those of the
+      Community and Business Group Process are void.
+    </p>
+    <p>
+      As with other Community Groups, W3C seeks organizational licensing
+      commitments under the <a href=
+      'https://www.w3.org/community/about/process/cla/'>W3C Community
+      Contributor License Agreement (CLA)</a>. When people request to
+      participate without representing their organization's legal interests,
+      W3C will in general approve those requests for this group with the
+      following understanding: W3C will seek and expect an organizational
+      commitment under the CLA starting with the individual's first request to
+      make a contribution to a group <a href="#deliverables">Deliverable</a>.
+      The section on <a href="#contrib">Contribution Mechanics</a> describes
+      how W3C expects to monitor these contribution requests.
+    </p>
+
+    <p>
+      The <a href="https://www.w3.org/Consortium/cepc/">W3C Code of
+      Ethics and Professional Conduct</a> applies to participation in
```

Comment:
```suggestion
      Conduct</a> applies to participation in
```
Update to latest (shorter) name.